packages/eslint-config-ndla/base.js

```diff
@@ -7,37 +7,5 @@ module.exports = {
     'import/no-cycle': ['warn', { maxDepth: Infinity }],
     'react/no-unused-state': 'warn',
     'react-hooks/rules-of-hooks': 'error',
-    'no-console': ['error', { allow: ['warn', 'error'] }],
```

Comment:
Mente du også å slette `no-console` linja? Vi la den til for å unngå å pushe `console.log` ved et uhell

Reply:
Tror ikke den linja har funka før, for jeg har aldri fått errors før ved bruk av console.log. Når det nå ble errors så ble det veldig irriterende under utvikling, så derfor tok jeg den bort

Reply:
Kan et alternativ være å sette den som 'warn' og ikke 'error'?

Reply:
Legger til det.